hat/assets/js/apps/Iaso/domains/app/components/SidebarMenuComponent.js

```diff
@@ -87,14 +87,10 @@ const styles = theme => ({
     },
 });
 
-const localizedManualUrl = (locale, account) => {
-    if (locale === 'fr' && account.name === 'RDC') {
-        return 'https://docs.google.com/document/d/1lKyhbKDLZpHtAsf3K6pRs0_EAXWdSDsL76Ohv0cyZQc/edit';
-    }
-
-    return account.user_manual_path
```

Comment:
@quang-le, the user_manual_path will not be used anymore? if yes why don't we just removed from the account?

Reply:
And on the line 200, the call the current.account parameter should be removed from the call of **localizedManualUrl**. 
![Screenshot from 2024-01-12 09-43-13](https://github.com/BLSQ/iaso/assets/19631540/0ddbbcad-c265-439a-9d5a-913eaefb474d)

Reply:
Mmmh, good comment. I guess, We should keep the thing configurable indeed.

Reply:
I've restored it and asked for an updated link to the rdc doc